.travis.yml

```diff
@@ -0,0 +1,29 @@
+dist: trusty
```

Comment:
Please rename this file to "travis.yml" and create symbolic link to this file named ".travis.yml"

Reply:
done